alpha_challenges/the_basics.md

```diff
@@ -153,5 +160,7 @@ end
 
 **Answer 12:**
 ```ruby
-# write your code here
+def caps(string)
+  puts string.upcase
```

Comment:
Same as Jackie mentioned. `puts` doesn't return anything. Ruby uses implicit returns and just returns the last line of the method. You could just write this instead:
```
def caps(string)
  string.upcase
end
```

Reply:
Got it! Thanks for the feedback!